dv/verilator/pcount/cpp/ibex_pcounts.cc

```diff
@@ -0,0 +1,57 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <numeric>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "ibex_pcounts.h"
+
+// see mhpmcounter_incr signals in rtl/ibex_cs_registers.sv for details
+
+const std::vector<std::string> ibex_pcount_names = {
+    "Cycles",
+    "NONE",
+    "Instructions Retired",
+    "LSU Busy",
+    "Fetch Wait",
+    "Loads",
+    "Stores",
+    "Jumps",
+    "Conditional Branches",
+    "Taken Conditional Branches",
+    "Compressed Instructions"};
+
+std::string ibex_pcount_string(uint64_t pcounts[], bool csv) {
+  char seperator = csv ? ',' : ':';
+  std::string::size_type longest_name_length;
+
+  if (!csv) {
+    longest_name_length = 0;
+    for(const std::string& pcount_name : ibex_pcount_names) {
```

Comment:
Should be `for (const std::string &pcount_name : ibex_pcount_names) {`. Now Philipp has merged a `.clang-format` file you can get clang-format to sort this for you. The current Debian/Ubuntu clang-format package includes the git-clang-format helper so you can stage your work and use `git clang-format` to reformat changed hunks.